trunk/build_project.sh

```diff
@@ -32,19 +32,27 @@ fi
 #  -DLLVM_ENABLE_RUNTIMES='compiler-rt' \
 #  -DLLVM_VERSION_SUFFIX=_$TRUNK_VERSION_STRING \
 #  -DCLANG_VENDOR=AMD_TRUNK_$TRUNK_VERSION_STRING \
+
+#  These options taken from build bot cause build fails.
+#  So for now we are not using them.
+#-DCMAKE_C_COMPILER_LAUNCHER=ccache
+#-DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
These options may fail because the `ccache` package must be installed first (`apt install ccache`). It's quite good for speeding up the compilation process, but it's not necessary.